vm/chef/cookbooks/dolibarr/files/dolibarr-setup-wizard

```diff
@@ -0,0 +1,134 @@
+cat << 'EOF' > /var/www/html/htdocs/install/install.forced.php
+
+<?php
+/* Copyright (C) 2016       Raphaël Doursenaud      <rdoursenaud@gpcsolutions.fr>
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+/** @var bool Hide PHP informations */
+$force_install_nophpinfo = true;
+
+/** @var int 1 = Lock and hide environment variables, 2 = Lock all set variables */
+$force_install_noedit = 2;
+
+/** @var string Information message */
+$force_install_message = 'Welcome to your Dolibarr install';
+
+/** @var string Data root absolute path (documents folder) */
+$force_install_main_data_root = '/var/lib/dolibarr/documents';
+
+/** @var bool Force HTTPS */
+$force_install_mainforcehttps = false;
+
+/** @var string Database name */
+$force_install_database = 'dolibarr';
+
+/** @var string Database driver (mysql|mysqli|pgsql|mssql|sqlite|sqlite3) */
+$force_install_type = 'mysqli';
+
+/** @var string Database tables prefix */
+$force_install_prefix = 'llx_';
+
+/** @var bool Force database creation */
+$force_install_createdatabase = false;
+
+/** @var string Database username */
+$force_install_databaselogin = '';
+
+/** @var string Database password */
+$force_install_databasepass = '';
+
+/** @var bool Force database user creation */
+$force_install_createuser = false;
+
+/** @var string Database root username */
+$force_install_databaserootlogin = 'root';
+
+/** @var string Database root password */
+$force_install_databaserootpass = '';
+
+/** @var bool Force install locking */
+$force_install_lockinstall = true;
+
+/** @var string Database server host */
+$force_install_dbserver = 'localhost';
+
+/** @var int Database server port */
+$force_install_port = 3306;
+
+$dolibarr_main_url_root = '';
+
+EOF
+
+# Replace variables in install.forced.php
+sed -i -e  "s|dolibarr_main_url_root =.*|dolibarr_main_url_root = '${main_url_root}';|g" /var/www/html/htdocs/install/install.forced.php
+
+sed -i -e "s|force_install_databaselogin =.*|force_install_databaselogin = '${db_username}';|g" /var/www/html/htdocs/install/install.forced.php
+
+sed -i -e "s|force_install_databasepass =.*|force_install_databasepass = '${db_password}';|g" /var/www/html/htdocs/install/install.forced.php
+
+# Start install wizard 
+curl -s -L http://localhost/install/check.php?testget=ok 
+
+curl -s -L -d 'selectlang=en_US' -d 'action=set' http://localhost/install/fileconf.php 
```

Comment:
How about alias for common options? imo actual steps are more readable this way

```
alias wizard_step='curl -s -L -d "selectlang=en_US" -d "action=set"'

wizard_step http://localhost/install/fileconf.php

wizard_step -d "main_url=http://$(get_external_ip)" http://localhost/install/step1.php

wizard_step http://localhost/install/step2.php

wizard_step http://localhost/install/step4.php

wizard_step -d "login=${admin_username}" -d "pass=${admin_password}" -d "pass_verif=${admin_password}" \
    http://localhost/install/step5.php
```

Naming up to you

Reply:
Yes, I think is a good idea to use an alias, I added the change in the script